lib/run-in-terminal.coffee

```diff
@@ -180,37 +180,55 @@ module.exports =
                 selectors.tree,
                 "tree-start-terminal-here",
                 "Start terminal here [tree]",
-                () => @tree_start_terminal_here(false)
+                () => @tree_start_terminal_here(false, false)
             ],
             [
                 selectors.tree,
                 "tree-start-terminal-here-and-run",
                 "Start terminal here and run [tree]",
-                () => @tree_start_terminal_here(true)
+                () => @tree_start_terminal_here(true, false)
+            ],
+            [
+                selectors.tree,
+                "tree-start-terminal-here-and-run-with-par",
+                "Start terminal here and run [tree] (with par)",
```

Comment:
@LevPasha 
I don't use context menu, but it looks like there will be no problems with sorting, since there is no sorting :smile: 
About naming. Maybe you are right… I think this can be toggled with something like «use short context names» option.